subworkflows/local/utils_nfcore_rnaseq_pipeline/main.nf

```diff
@@ -256,6 +256,33 @@ def validateInputParameters() {
         }
     }
 
+    // Check that unaligned reads are saved when using Kraken2
+    if (params.kraken_db) {
```

Comment:
My bad. I did not look closely enough into which file you put it and somehow believed you made a local subworkflow for the contamination screening that would be run later. Since you put it into the _utils_ subworkflow, it is already at the perfect spot! Thanks and sorry for the fuss.